util/stmtsummary/evicted.go

```diff
@@ -51,6 +55,22 @@ func newStmtSummaryByDigestEvictedElement(beginTime int64, endTime int64) *stmtS
 		beginTime:    beginTime,
 		endTime:      endTime,
 		digestKeyMap: make(map[string]struct{}),
+		detail: &stmtSummaryByDigestElement{
+			beginTime: beginTime,
+			endTime:   endTime,
+			// basic
+			sampleSQL:  "other",
+			prevSQL:    "other",
+			indexNames: []string{"other"},
```

Comment:
```suggestion
```

Reply:
removing this will fail the test in statement_summary_test.go:618: testStmtSummarySuite.TestToDatum()